docs/api.rst

```diff
@@ -3705,6 +3705,14 @@ PartialInteractionMessage
     :members:
     :inherited-members:
 
+MessageInteraction
+~~~~~~~~~~~~
```

Comment:
Should match the length
```suggestion
~~~~~~~~~~~~~~~~~~
```

Reply:
Should I re-request review in charge requests? Or just wait